### DISCUSSION TITLE
采用本地部署方案邮件通知发送失败

### DISCUSSION THREAD

**Discussion by elkan1788** [walinejs/waline#1320] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/1217096/187034528-a49c50c0-770b-4ae0-a7c3-bd263583b750.png)

如图上所示，发送邮件通知服务失败！

另外想了解下如何配置那个默认`8360`端口为其他端口呢？

**Comment by lizheming** [comment, marked as answer]
`No recipients defined` 未指定收件人，可以看下评论数据邮箱之类的是否有。

**Reply by elkan1788** [reply]
已经检查过了，确认是有收件人的邮箱信息，不清楚为何会出这样的错误。
![image](https://user-images.githubusercontent.com/1217096/187053423-03853d27-8bd4-420a-8c3e-417a4651d044.png)

**Reply by elkan1788** [reply]
看起来是应该是个:bug:， 上面虽然是报错，但邮件是可以成功发出去的

部署到远程服务器还是一样的错误
![image](https://user-images.githubusercontent.com/1217096/187054734-ef1d1349-5316-4fe7-9fa5-2dd1d787e719.png)
邮件发送成功
![image](https://user-images.githubusercontent.com/1217096/187054747-6b02026f-8b70-4890-92bc-e8e05251d3e7.png)

**Reply by lizheming** [reply]
是不是没指定 `AUTOR_MAIL` 博主邮箱，应该是博主通知的报错

**Reply by elkan1788** [reply]
还真的是呀，有点郁闷了，现在问题解决了，谢谢。
